core/src/main/java/org/apache/carbondata/core/datastore/page/ColumnPage.java

```diff
@@ -206,6 +199,7 @@ public static ColumnPage newPage(TableSpec.ColumnSpec columnSpec, DataType dataT
       }
     } else {
       switch (dataType) {
+        case BOOLEAN:
```

Comment:
It is better to add a `newBooleanPage`, otherwise `newBytePage` internal will store the data type as byte instead of boolean